packages/react/src/lib/accessibility/FocusZone/CHANGELOG.md

```diff
@@ -26,6 +26,12 @@ This is a list of changes made to this Stardust copy of FocusZone in comparison
 - Handle keyDownCapture based on `shouldHandleKeyDownCapture` prop @sophieH29 ([#563](https://github.com/stardust-ui/react/pull/563))
 - Add `bidirectionalDomOrder` direction allowing arrow keys navigation following DOM order @sophieH29 ([#1637](https://github.com/stardust-ui/react/pull/1647))
 
+### Upgrade `FocusZone` to the latest version from `fabric-ui` @sophieH29 ([#1772](https://github.com/stardust-ui/react/pull/1772))
+- Restore focus on removing item ([OfficeDev/office-ui-fabric-react#7818](https://github.com/OfficeDev/office-ui-fabric-react/pull/7818))
+- Reduce global event listeners ([OfficeDev/office-ui-fabric-react#7958](https://github.com/OfficeDev/office-ui-fabric-react/pull/7958))
+- Track innerzones correctly ([OfficeDev/office-ui-fabric-react#8560](https://github.com/OfficeDev/office-ui-fabric-react/pull/8560))
```

Comment:
No UTs from 8560.

Reply:
I didn't like an idea to add a public static method to FZ API to test it as here https://github.com/OfficeDev/office-ui-fabric-react/pull/8560/files#diff-2949b523d6a4d1f7e2e111abb6557158R82
If you're ok with that, I'll add it

Reply:
I would rather add it